package.json

```diff
@@ -48,12 +48,12 @@
     "@babel/eslint-parser": "^7.25.7",
     "@types/express": "^5.0.0",
     "@webpack-cli/serve": "^2.0.4",
-    "eslint": "^8.57.1",
-    "eslint-config-standard": "^17.1.0",
+    "eslint": "^9.11.1",
     "eslint-plugin-import": "^2.31.0",
-    "eslint-plugin-n": "^16.6.2",
-    "eslint-plugin-promise": "^6.6.0",
+    "eslint-plugin-n": "^17.10.3",
+    "eslint-plugin-promise": "^7.1.0",
```

Comment:
Neostandard bundles these plugins, you do not need and shouldn't really include them yourself anymore 🙂